lib/omnibus/packager.rb

```diff
@@ -48,6 +49,7 @@ module Packager
       "ips"      => IPS,
       "windows"  => [MSI, APPX],
       "mac_os_x" => PKG,
+      "freebsd"  => PKGNG,
```

Comment:
To ensure some backward compat let's produce both a `Makeself` and `PKGNG` artifact on FreeBSD. The following code should work (be sure to update the specs also):

```ruby
"freebsd"  => [Makeself, PKGNG],
```